console/src/config.rs

```diff
@@ -176,3 +206,16 @@ fn parse_true_color(s: &str) -> bool {
     let s = s.trim();
     s.eq_ignore_ascii_case("truecolor") || s.eq_ignore_ascii_case("24bit")
 }
+
+impl FromStr for RetainFor {
+    type Err = humantime::DurationError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "none" => Ok(RetainFor(None)),
```

Comment:
i think we may want to do this instead:
```suggestion
            s if s.eq_ignore_ascii_case("none") => Ok(RetainFor(None)),
```
so that the user can also write "None"...